### ISSUE TITLE
Cannot find module 'prop-types' at startup

### ISSUE THREAD

**Issue by jhogins** [Hashnode/mern-starter#317]
Ran for the first time and got "Error: Cannot find module 'prop-types'"

**Comment by Nicholson85** [comment]
I don't believe 'prop-types' is an included package for React v 15.5.3 > and must now be installed separately.

**Comment by infacq** [comment]
I'm planning to use this starter package but saw the repo has long time no update while upgrading to new React might take sometime especially when dealing with `createClass`
